internal/v2/fakes2av2/fakes2av2.go

```diff
@@ -213,8 +216,25 @@ func validatePeerCertificateChain(req *s2av2pb.ValidatePeerCertificateChainReq)
 }
 
 // TODO(rmehta19): Update this to return ciphersuites in Client/Server TlsConfiguration.
```

Comment:
Should this comment be deleted?

Reply:
I think so yes, good catch. I'm going to hold off doing so here because I should do a careful pass of Riya's TODOs and see what (if anything) needs to be done. Will send you a follow-up PR for this.